lnrpc/lightning.proto

```diff
@@ -3711,6 +3718,22 @@ message ListPermissionsResponse {
     map<string, MacaroonPermissionList> method_permissions = 1;
 }
 
+message NodeAnnouncementUpdateRequest {
+    // Features holds the set of features the node supports.
+    map<uint32, Feature> features = 1;
```

Comment:
So we can use the existing `FeatureBit` enum here: https://github.com/lightningnetwork/lnd/blob/master/lnrpc/lightning.proto#L3407